controllers/feed.js

```diff
@@ -227,11 +227,24 @@ exports.deletePost = async (req, res, next) => {
     // Delete post in Database
     const deletePostById = await Post.findByIdAndRemove(postId);
 
+    // Get user from Database
+    const getUserResult = await User.findById(req.userId);
+
+    // Clear post-user relations
+    await getUserResult.posts.pull(postId);
```

Comment:
If cut `await` keyword in front of `getUserResult` the application still working